armi/bookkeeping/db/database3.py

```diff
@@ -912,7 +1077,16 @@ def _writeParams(self, h5group, comps):
                     attrs[_SERIALIZER_NAME] = paramDef.serializer.__name__
                     attrs[_SERIALIZER_VERSION] = paramDef.serializer.version
                 else:
```

Comment:
Note: we are almost always ending up in this `else` block. Looking through all of the parameter definitions in the framework, the only one that specifies a `Serializer` is `flags`:

https://github.com/terrapower/armi/blob/8cfc244ced1767bf71376d41796c618165c61bb7/armi/reactor/composites.py#L60

Reply:
Yup, that sounds right.

Though we might be adding some more serializers for this ticket:  https://github.com/terrapower/armi/discussions/1688